hooks/useRewards.ts

```diff
@@ -0,0 +1,43 @@
+import { CosmWasmClient } from "@cosmjs/cosmwasm-stargate"
+import { useQuery } from "react-query"
+import { useRecoilValue } from "recoil"
+import { getClaimableRewards, getRewardsInfo } from "services/rewards"
+import { walletState, WalletStatusType } from "state/atoms/walletAtoms"
+import { DEFAULT_TOKEN_BALANCE_REFETCH_INTERVAL } from "util/constants"
+import { convertMicroDenomToDenom } from "util/conversion"
+
+export const useClaimableRewardsBalance = (rewardsAddress: string, decimals: number) => {
+  const { address, status, client } = useRecoilValue(walletState)
+
+  const { data: rewards = 0, isLoading } = useQuery(
+    [`claimableRewards/${rewardsAddress}`, address],
+    async () => {
+      return convertMicroDenomToDenom(await getClaimableRewards(address,rewardsAddress,client), decimals)
+    },
+    {
+      enabled: Boolean(rewardsAddress && status === WalletStatusType.connected),
+      refetchOnMount: 'always',
+      refetchInterval: DEFAULT_TOKEN_BALANCE_REFETCH_INTERVAL,
+      refetchIntervalInBackground: true,
+    }
+  )
+
+  return { rewards, isLoading }
```

Comment:
minor nit: think you can return an array here? 
```suggestion
  return [rewards, isLoading]
```

that way you can have anything for the variables name later on:
```ts
const [rewardsBalance, fetchingBalance] = useClaimableRewardsBalance()
```